All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+# EIPIP Meeting 11 Notes
+### Meeting Date/Time: Wednesday 1 July 2020 at 15:00 UTC
+### Meeting Duration: 60 minutes
+### [GitHub Agenda Page](https://github.com/ethereum-cat-herders/EIPIP/issues/21)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=wZnsVp0aqBk)
+### Moderator: James Hancock
+### Notes: Jim Bennett
+
+## Summary
+
+## Decisions Made:
+
+Decision Item | Description
+-|-
+**11.1** | Closed the issue on documenting the Cat Herders process
+
+## Actions Required:
+
+Action Item | Description
+-|-
+ **11.1**        | Introduce EIP 2046 in a future AllCoreDevs call to be considered as an EFI.
+**11.2**        | Discuss an emergency communications checklist on the next EIPIP call.
+**11.3**        | Pooja to add Jaye Harill to the comms group in the Cat Herders
+**11.4**        | BFollow up with lightclient on his new validator and the rules.
+**11.5**        | Read [this link](https://www.rfc-editor.org/about/independent/#:~:text=To%20post%20an%20Internet%2DDraft,%40rfc%2Deditor.org.) about independent submissions as part of the RFC before the next meeting.
+**11.6**       | James to write the precompile discussion item into a PR.
+**11.7**       | James to work on doing a document or a PR for the statuses to EIP 1
+**11.8**       | Look at the Cat Herders to investigate the project board for the EIPIP repo.
```

Comment:
Investigate the Cat Herders repo to use the project board for tracking the EIPIP agenda items.